### ISSUE TITLE
Not working with PostCSS 8 / Webpack 4 / TailwindCSS

### ISSUE THREAD

**Issue by lewebsimple** [limitlessloop/flex-gap-polyfill#19]
I can't get this to work with the latest PostCSS and Webpack 4.

postcss.config.js
```
module.exports = {
  plugins: [
    require('tailwindcss'),
    require('postcss-preset-env'),
    require('flex-gap-polyfill')(),
  ],
};
```

I get the following error:
```
JisonLexerError: Lexical error on line 1: Unrecognized text.

  Erroneous area:
1: -auto * var(--fgp-gap_percentage-decimal_row)
^...^
```

**Comment by limitlessloop** [comment]
Hi, thanks for raising this issue. Admittedly I've not had chance to try this with PostCSS 8. I'll investigate this issue shortly.

**Comment by limitlessloop** [comment]
@lewebsimple I just want to check something. Are you by any chance using `margin: auto;` with the `gap' property in any way? This might be a bug/limitation which I'm recently become aware of but not had chance to make better. It shouldn't be causing it to error but PostCSS 8 might have some stricter linting rules. I will investigate it further when I get chance. For now if it is related to that, you might have to use `margin: auto` on a separate wrapper container if possible.

**Comment by limitlessloop** [comment]
Hi @yoanmalie. Thanks for this. I'll take a look at it this weekend. I think the issue might be being caused by the use of percentage gaps and possibly the container has a width of `auto`.  The polyfill uses the width set on the container to help aid in calculating the correct gap spacing. It might be in this case that a fallback might be required, but I'll investigate it further at the weekend or the end of this week.

**Comment by limitlessloop** [comment]
Ok I think I know what the issue is, hoping to have a fix ready by the end of the week.

Basically the plugin is generating invalid CSS in certain scenarios when `width: auto' exists in the stylesheet but I didn't spot this before PostCSS 8.

**Comment by limitlessloop** [comment]
Hi @yoanmalie @lewebsimple I've published a minor fix which I hope will resolve the error you're seeing. Please update to version 1.0.4 and let me know if it resolves the issue you are experiencing.

**Comment by limitlessloop** [comment]
Closing this issue now as should be fixed in `2.0.1`.
